examples/tensorflow/tf1/README.md

```diff
@@ -0,0 +1,55 @@
+## MLflow with TensorFlow 1.X
+
+In this example, we train a `tf.estimator.DNNRegressor` on the TensorFlow [Boston Housing dataset](https://www.cs.toronto.edu/~delve/data/boston/bostonDetail.html).
+With MLflow, we use `mlflow.tensorflow.autolog()` along with `mlflow.log_param()` and `mlflow.log_metric()` in order to capture data about how we trained the target model.
```

Comment:
```suggestion
The code is mostly pure TensorFlow - we add a call to `mlflow.tensorflow.autolog()` before training to record params & metrics from model training (e.g. model loss) as part of an MLflow run. After training, we save the fitted model to the same MLflow run via `mlflow.tensorflow.log_model`, allowing us to associate the model with training metrics & params. We then demonstrate how to load the saved model back as a generic `mlflow.pyfunc`, allowing us to make predictions on pandas DataFrames
```

Reply:
The code doesn't actually call `log_model`, only `classifier.export_saved_model` (which autolog patches over and calls from `log_model` from there). Should we revise the second sentence to reflect this?